handbook/marketing/content/web-series.md

```diff
@@ -0,0 +1,71 @@
+## Web series
+
+Describe what a web series is and why we do them at Sourcegraph
```

Comment:
Is this a left-over TODO or still valid TODO? If it's the latter I'd hide it in a comment:

```suggestion
<!-- Describe what a web series is and why we do them at Sourcegraph -->
```